apistructs/extention.go

```diff
@@ -207,7 +224,7 @@ type ExtensionVersionGetRequest struct {
 // ExtensionVersionQueryRequest 查询extension版本
 type ExtensionVersionQueryRequest struct {
 	Name       string
-	YamlFormat bool
+	YamlFormat bool `query:"yamlFormat"`
```

Comment:
why just add query tag for this field? how about others fields?

Reply:
other fileds doesn't existed in query